Fabric.Identity.API/scripts/Install-Identity.ps1

```diff
@@ -68,6 +68,14 @@ $installApplication = Publish-Application -site $selectedSite `
                  -zipPackage $zipPackage `
                  -assembly "Fabric.Identity.API.dll"
 
+# Get Idpss app pool user 
+# Create log directory with read/write permissions for app pool user
+$idpssName = "IdentityProviderSearchService"
+$idpssAppPoolUser = Find-IISAppPoolUser -applicationName $idpssName -discoveryServiceUrl $discoveryServiceUrl -noDiscoveryService $noDiscoveryService -quiet $quiet
+$idpssDirectory = [io.path]::combine([System.Environment]::ExpandEnvironmentVariables($selectedSite.physicalPath), $idpssName)
+New-AppRoot $idpssDirectory $idpssAppPoolUser
```

Comment:
this justs grant access to the root and all contained folders (including the logs folder and anything else we do in the future?)

Reply:
i see now that the new-appRoot takes care of the log directory!